firebase-inappmessaging/src/main/java/com/google/firebase/inappmessaging/model/CardMessage.java

```diff
@@ -148,8 +148,8 @@ public Action getSecondaryAction() {
   }
 
   /**
-   * @deprecated Use getPrimaryAction() instead. Gets the primary {@link Action} associated with
-   *     this message
+   * @deprecated This method is deprecated. Use {@link #getPrimaryAction()} or {@link
```

Comment:
I _think_ that by having the @deprecated that the "This method is deprecated." bit will automatically get generated when we generate the docs.